src/extensions/email_previews.cr

```diff
@@ -0,0 +1,10 @@
+# TODO: Move this in to Carbon directly
+abstract class Carbon::EmailPreviews
+  alias EmailLookup = Hash(String, Carbon::Email)
```

Comment:
Maybe this should be `Symbol` instead of `String` because this gets passed through the URL, and it would break if someone made an email key named "Welcome Email". 🤔

Reply:
Nope.. The key comes through the path params, and you can't turn a String in to a Symbol.. So this has to be a String. Does this mean we should validate it? Or should we just do a URI encode/decode ?